### ISSUE TITLE
Unable to send email through google cloud. The error is caused by EHLO message

### ISSUE THREAD

**Issue by mariachimania** [Mailtrain-org/mailtrain#1002]
Wireshark capture shows the following:

220 smtp-relay.gmail.com ESMTP p5sm1520525phu.3 - gsmtp
EHLO [127.0.0.1]
421 4.7.0 Try again later, closing connection. (EHLO) p5sm1520525phu.3 - gsmtp

There's no option to edit the loopback address on the settings.

**Comment by mariachimania** [comment]
The code is suppose to get the correct hostname from somewhere in my server but it wasn't working. I ended up modifying the nodemailer files to force the hostname I wanted. I had to edit the following file:  /mailtrain/node_modules/nodemailer/lib/smtp-connection/index.js

The file ended up like this:

_getHostname() {
        // defaul hostname is machine hostname or [IP]
        let defaultHostname = '[subdomain1.maindomain.com]';
        return defaultHostname;
    }

However I couldn't fix the problem on mailtrain running on docker. I couldn't find my way into this folder inside docker.
I hope this post helps.
